### ISSUE TITLE
Angular module not available

### ISSUE THREAD

**Issue by abroekhuis** [ruddell/generator-jhipster-ionic#27]
When creating a new project based on a new jhipster project, the generator uses "main" as module name, whereas the jhipster project has a custom name for the module.
Now when copying over the jhipster files, there is an unknown module, since the app.module.js isn't copied over.
I've now added a new module in the jhipster directory without any dependencies to be able to use the jhipster files. But I think the generator should solve this. Can it inspect the app.module.js and use the module name specified in there?

**Comment by ruddell** [comment]
Good idea and thanks for the feedback.  Just to confirm, you want the components in JHipster's app.module.js to be imported into main.js [here](https://github.com/ruddell/generator-jhipster-ionic/blob/master/generators/app/templates/m-ionic/_main.js#L15)?
